Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/APIHelpers/AllShowsAPIHelper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AllShowsModel.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Phoenix McKnight on 9/10/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import Foundation
+
+class AllShowsAPIHelper {
+    static let shared = AllShowsAPIHelper()
+    private init () {}
+    
+    
+    
```

Comment:
I love that you used an optional string in the getShow func. This way you have a default url if no string is provided in the search bar. Great job.